frontend/src/instance-views/ComparisonView.svelte

```diff
@@ -124,58 +180,116 @@
 
 		await tick();
 
-		[$model, ...$comparisonModels].forEach((mod) => {
+		[$model, $comparisonModel].forEach((mod) => {
 			let obj = $status.completeColumns.find((c) => {
 				return c.columnType === ZenoColumnType.OUTPUT && c.model === mod;
 			});
 			let modelColumn = obj ? columnHash(obj) : "";
-
-			tables[mod].forEach((_, i) => {
-				let div = viewDivs[mod][i];
-				if (div) {
-					viewFunction(
-						div,
-						viewOptions,
-						tables[mod][i],
-						modelColumn,
-						columnHash($settings.labelColumn),
-						columnHash($settings.dataColumn),
-						idHash
-					);
-				}
-			});
+			if (tables[mod] && viewDivs[mod]) {
+				tables[mod].forEach((_, i) => {
+					let div = viewDivs[mod][i];
+					if (div) {
+						viewFunction(
+							div,
+							viewOptions,
+							tables[mod][i],
+							modelColumn,
+							columnHash($settings.labelColumn),
+							columnHash($settings.dataColumn),
+							idHash
+						);
+					}
+				});
+			}
 		});
 	}
 </script>
 
 <Svelecte
```

Comment:
Can we add text to the left of this saying "comparison feature" or something like this

Reply:
Also add margin-right to line up with the end of the table on the right

Reply:
> Also add margin-right to line up with the end of the table on the right

hmm not really get the idea... should the align look something like this? 
![image](https://github.com/zeno-ml/zeno/assets/89871868/41104986-71b9-4b0f-bfdc-074e97e713e9)

Reply:
Yup! right margin can be smaller though, just to align with the right side where the other stuff ends like the end of the refresh button

Reply:
hmmm, does my view looks different from yours? 
This is the original view and I think it's pretty aligned with the right side.

![image](https://github.com/zeno-ml/zeno/assets/89871868/88bceb65-0aee-4243-bd53-1846b7677876)

Reply:
Looks fine!